gamestonk_terminal/mutual_funds/investpy_view.py

```diff
@@ -136,19 +142,22 @@ def display_historical(data: pd.DataFrame, fund: str = "", export: str = ""):
         Fund symbol or name
     export: str
         Format to export data
+    external_axes:Optional[List[plt.Axes]]:
+        External axes to plot on
     """
-    console.print("")
-    fig, ax = plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
-    ax.plot(data.index, data.Close, "-g")
-    ax.grid("on")
+    console.print()
+    if not external_axes:
+        _, ax = plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
+    else:
+        ax = external_axes[0]
+    ax.plot(data.index, data.Close)
     ax.set_xlim([data.index[0], data.index[-1]])
     ax.set_xlabel("Date")
     ax.set_ylabel("Close Price")
     ax.set_title(f"{fund.title()} Price History")
-    fig.autofmt_xdate()
-    fig.tight_layout(pad=1)
-    if gtff.USE_ION:
-        plt.ion()
-    plt.show()
+    ax.yaxis.set_label_position("right")
```

Comment:
this is set with other style stuff in `theme.style_primary_axis`, so no need to have it in the view